tests/test_handers_render.py

```diff
@@ -0,0 +1,97 @@
+"""The module contains the tests for the render config which is generated in handlers."""
+
+from hammett.core import Screen
+from hammett.core.constants import FinalRenderConfig
+from hammett.test.base import BaseTestCase
+from hammett.test.utils import catch_render_config
+from tests.base import BaseTestScreenWithDescription, TestScreen, TestStartScreen
+
+
+class BaseTestScreenWithDynamicDescription(Screen):
+    """The class represents the base screen with dynamic description for the testing purposes."""
+
+    DYNAMIC_DESCRIPTION = 'Test dynamic description'
+
+    async def get_description(self, _update, _context) -> str:
+        """Return the `DYNAMIC_DESCRIPTION` attribute of the screen."""
+        return self.DYNAMIC_DESCRIPTION
+
+
+class TestScreenWithDynamicDescription(BaseTestScreenWithDynamicDescription):
+    """The class implements a screen for the tests with dynamic description."""
+
+
+class TestScreenWithDynamicAndStaticDescription(
+    BaseTestScreenWithDescription,
+    BaseTestScreenWithDynamicDescription,
+):
+    """The class implements a screen for the tests with dynamic and static description."""
+
+
+class HandlersRenderTests(BaseTestCase):
+    """The class implements tests for the render config which is generated in handlers."""
+
+    @catch_render_config()
+    async def test_screen_jump_handler_render(self, catcher):
```

Comment:
I suggest we rename `catcher` to `actual` so that the assert line below looks like

```
self.assertFinalRenderConfigEqual(expected, actual.final_render_config)
```

Reply:
Anyway, `catcher` isn't suitable, as it doesn't represent the catcher itself, but rather the result of its operation, i.e. the render config.`